docs/src/main/paradox/client-side/client-transport.md

```diff
@@ -21,23 +21,10 @@ transport layer itself.
 
 ## Configuring Client Transports
 
-A @unidoc[ClientTransport] is configured slightly differently for the various layers of the HTTP client.
-Right now, configuration is only possible with code (and not through config files). There's currently no
-predefined way that would allow you to select different transports per target host (but you can easily define any kind
-of strategy by implementing @unidoc[ClientTransport] yourself).
-
-### Connection Pool Usage
-
-The @unidoc[ConnectionPoolSettings] class allows setting a custom transport for any of the pool methods. Use
-`ConnectionPoolSettings.withTransport` to configure a transport and pass those settings to one of the
-pool methods like
-@scala[`Http().singleRequest`, `Http().superPool`, or `Http().cachedHostConnectionPool`]
-@java[`Http.get(...).singleRequest`, `Http.get(...).superPool`, or `Http.get(...).cachedHostConnectionPool`].
-
-### Single Connection Usage
-
-You can configure a custom transport for a single HTTP connection by passing it to the `Http().outgoingConnectionUsingTransport`
-method.
+A @unidoc[ClientTransport] can be configured in the @unidoc[ClientConnectionSettings]. Right now, this is not possible
```

Comment:
I simplified the docs. There's only one way to configure the transport which is always through `ClientConnectionSettings` which can be passed into all of the `Http` client-side methods. That's the great simplification of this PR.